src/app/products.php

```diff
@@ -1,20 +1,70 @@
-<?php
-
-namespace store;
 
-use \PDO;
-use \store\Product;
+<?php
 
-$db = new PDO('mysql:host=127.0.0.1; dbname=ecommerceStore2018', 'root');
-$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+require_once '../../vendor/autoload.php';
+use Store\Category as Category;
+use Store\Product as Product;
+use Store\DBConnect as DBConnect;
+use Store\Store as Store;
 
-$query = $db->prepare("SELECT * FROM products WHERE categoryId = :categoryId");
+$dBConnect = DBConnect::connectToDB();
+Store::setPDO($dBConnect);
+$categories = Store::getCategories();
+$products = Store::getProducts($_GET['categoryId']);
 
-$categoryId = '1';
-//var_dump($_GET['id']);
+?>
 
-$query->bindParam(':categoryId', $categoryId);
-$query->execute();
-$result = $query->fetchAll(PDO::FETCH_CLASS, 'store/Product');
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Totally Pawesome</title>
+    <link rel="stylesheet" type="text/css" href="../assets/css/vendor/bootstrap.min.css">
+    <link rel="stylesheet" type="text/css" href="../assets/css/styles.css">
+</head>
+<body>
+<section class="row banner">
+    <div class="home col-xs-3 col-sm-2">
+        <a href="#">
+            <h2>Home</h2>
+        </a>
+    </div>
+    <div class="shop-name col-xs-9 col-sm-10">
+        <h2>Totally Pawesome</h2>
+    </div>
+</section>
+<section class="row main">
+    <div class="sidebar col-xs-3 col-sm-2">
+        <?php
+        foreach($categories as $category) {
+            if($category instanceof Category) {
+                echo $category->getCategoryListLink();
+            }
+        }
+        ?>
+    </div>
+    <div class="main-content col-xs-9 col-sm-10">
+        <div class="row">
+            <h2>Product Category</h2>
+        </div>
 
-var_dump($result);
\ No newline at end of file
+            <?php
+            foreach($products as $product) {
+                if($product instanceof Product) { ?>
```

Comment:
on line 41 a similar action was achieved but instead of hard-coding the HTML it was placed into a method that was called. Not sure if either is better, but we probably need to be consistent with our approach